tests/src/JIT/Directed/tailcall/more_tailcalls.cs

```diff
@@ -0,0 +1,998 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using InlineIL;
```

Comment:
Do you mind also adding a small readme into this folder explaining that this file is not used by testing, it is here to help regenerate the IL test.

Reply:
Added a comment to the top of the file